src/view/components/cpx/Cpx_svg_style.tsx

```diff
@@ -141,6 +141,10 @@ export const SVG_STYLE = `
         fill:#fff;
         pointer-events: none; user-select: none;
     }
+    .sim-text-outside{
+        font-size:8px;
+        fill: var(--vscode-descriptionForeground);
+    }
```

Comment:
It seems that we have some overlapping CSS code between the CPX and micro:bit. Might be worth considering sharing it between the components.

Reply:
Yes, right now the way it is applied on cpx and microbit is different, this can be done if we refactor cpx. (the microbit implementation is more solid as well, so it will be worth doing it)